lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1712,9 +1764,15 @@ type instance ApiAddressIdT (n :: NetworkDiscriminant) =
 type instance ApiCoinSelectionT (n :: NetworkDiscriminant) =
     ApiCoinSelection n
 
+type instance ApiCoinSelectionCertT (n :: NetworkDiscriminant) =
+    ApiCoinSelectionCert n
+
 type instance ApiSelectCoinsDataT (n :: NetworkDiscriminant) =
     ApiSelectCoinsData n
 
+type instance ApiSelectCoinsActionDataT (n :: NetworkDiscriminant) =
+    ApiSelectCoinsActionData n
```

Comment:
I don't think we need such type instances here. We need those for data-types which carry _addresses_, because of the type-level network discriminant. But if the underlying data-type do not contain any addresses, then this is superfluous / more complex than necessary.